frontend/www/js/omegaup/components/course/AddStudents.vue

```diff
@@ -80,6 +80,11 @@
           </tr>
         </tbody>
       </table>
+      <div class="float-right">
+        <a class="btn btn-primary" :href="studentsProgressUrl()">{{
```

Comment:
Qué te parece si aquí utilizas una propiedad computada en lugar de un método:
```suggestion
        <a class="btn btn-primary" :href="studentsProgressUrl">{{
```
De esta forma el resultado sólo se evalúa una vez, ya que no hay ninguna propiedad que modifique su valor:

https://digitalya.co/blog/vue-js-methods/#:~:text=A%20computed%20property%20is%20cached,changed%20inside%20the%20method%20property.